pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -326,6 +480,9 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *caOptions) (
 	if err != nil {
 		return nil, fmt.Errorf("failed to create an istiod CA: %v", err)
 	}
+
+	s.initCACertsWatcher()
```

Comment:
would be good to gate this with a PILOT flag, until we have e2e tests for this. Flag default should be off. Refer pilot/pkg/features/pilot.go.

Reply:
ok, good idea.